packages/utils/vite.config.ts

```diff
@@ -14,7 +14,4 @@ export default defineConfig({
       formats: ['es'],
     },
   },
-  define: {
-    'process.env': process.env,
-  },
```

Comment:
@colin-welch ran into a problem removing this from the console in Oystehr. @saewitz does npm run build work?

Reply:
Ottehr already uses `import.meta` and compatible TypeScript settings

Reply:
great!